### ISSUE TITLE
plone.restapi 6.4.1 shows "Volto Blocks" installer in add-on products controlpanel

### ISSUE THREAD

**Issue by erral** [plone/plone.restapi#895]
I have added plone.restapi 6.4.1 to a Plone 4.3.x buildout, and after fighting buildout, I see "Volto Blocks" profile as installable in add-on products control panel

![2020-03-30 13-24-15-ko-pantaila-argazkia](https://user-images.githubusercontent.com/817365/77907351-c89c6480-7289-11ea-9e70-7b08d066d230.png)

I expected to see the default plone.restapi profile.

I guess that in Plone 4.3.x Plone shows the alphabetically first profile, and "blocks" goes before "default".

**Comment by tisto** [comment]
@erral yeah. that's a known issue that has indeed nothing to do with plone.restapi. hard to find a solution for this...

**Comment by avoinea** [comment]
I think renaming this https://github.com/plone/plone.restapi/blob/864d56637c0b2d1c2b9cab50228c5ce63f58db5b/src/plone/restapi/configure.zcml#L50 to something like `vblocks` or `volto-blocks` would fix it.

**Comment by erral** [comment]
Yes, the vblocks would fix it, in essence anything that alphabetically goes after "default" would do it.

And is necessary to be able to install plone.restapi in Plone 4.3.x.

**Comment by tisto** [comment]
I'd be ok with a rename to "volto-blocks". The question would be is that a breaking change and do we need an upgrade step for this then?

**Comment by erral** [comment]
When do we need to install the "blocks" profile?

**Comment by avoinea** [comment]
From what I can see it just adds the `volto.blocks` behavior on `Page`.

Renaming it will break this:

```
<dependency>profile-plone.restapi:blocks</dependency>
```

within other packages `metadata.xml`

Respectively:
* `kitconcept.volto`
* `kitconcept.voltodemo`
* `eea.restapi`

See https://github.com/search?q=profile-plone.restapi%3Ablocks&type=Code

**Comment by wesleybl** [comment]
@avoinea @tisto can we change the name? The other packages can be upgraded as well.

**Comment by avoinea** [comment]
@wesleybl :+1:

**Comment by tisto** [comment]
I am not sure if it is worth the hassle to change the names now. If we start with Plone 6 we might have to refactor things anyways.

**Comment by erral** [comment]
I would do the name change and release one final plone.restapi version for Plone 4.3.x with the name changed. This way all users wanting to use plone.restapi in Plone 4.3 will have a reference version that can be installed without issues through Plone Control Panel. 

And then all efforts can go to Plone 5/6, with the wanted profile names or new features.

**Comment by tisto** [comment]
If we remove that profile altogether we won't have that problem and all future versions of p.restapi can be used with Plone 4.

**Comment by erral** [comment]
That would be OK if we are not using that profile anywhere

**Comment by wesleybl** [comment]
> If we remove that profile altogether we won't have that problem and all future versions of p.restapi can be used with Plone 4.

@tisto is your idea to transfer the behavior of this profile to the default profile and remove the profile? Wouldn't that put unnecessary behavior on those who don't use Volto?

**Comment by tisto** [comment]
@wesleybl no. Just move the Volto profile to another package. Those are important architectural decisions we have to make when moving towards Plone 6.

**Comment by wesleybl** [comment]
@tisto Well, it looks like it will take time. I would like to see this problem resolved in the next release of plone.restapi. Can we change the name of the profile? I can test what happens, on portals that have the current profile installed. There will probably be no problem.

**Comment by wesleybl** [comment]
Fix in https://github.com/plone/plone.restapi/commit/597d10d9983c1fff6024b2f36f352d10e291cb81
